lib/src/revset.rs

```diff
@@ -2415,7 +2415,11 @@ pub trait Revset: fmt::Debug {
 
     fn is_empty(&self) -> bool;
 
-    fn count(&self) -> usize;
+    /// Inclusive lower bound and, optionally, inclusive upper bound of how many
+    /// commits are in the revset, computed in O(1). The implementation may
```

Comment:
nit: drop the "computed in O(1)" since we don't promise that. The sentence after should also be updated or deleted.

Reply:
Ah, yes. Done.